anime_tracker/src/components/AppBar.vue

```diff
@@ -1,26 +1,44 @@
 <script lang="ts">
 
-import { defineComponent } from 'vue';
+import { defineComponent, ref } from 'vue';
 
-  export default defineComponent({
-    setup() {
+export default defineComponent({
+  props: {
+    searchQuery: String,
+  },
+  emits: [ "onSearch" ],
+  setup(props, { emit }) {
+    const searchField = ref("");
 
-    },
-    components:{
+    const handleSearch = () => {
+      emit('onSearch', searchField.value);
+    };
 
-    }
-  })
+    return {
+      searchField,
+      handleSearch,
+    };
+  },
+});
 </script>
 <template>
   <v-app-bar
           color="#C08497">
```

Comment:
avoid hard coded styles as much as possible. especially font and color which you possibly need multiple times -> make a variable, that you can change it on a single location

Reply:
is it the best case to use the sass variables?